### PR TITLE
[Merged by Bors] - Version bump `milagro_bls` to v1.4.2

### PR DESCRIPTION
**Pull request by kirk-baird**
## Issue Addressed

n/a

## Proposed Changes

Bump the version of `milagro_bls` to version v1.4.2

## Additional Info

No breaking changes.

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -1632,8 +1632,7 @@ checksum = "212d0f5754cb6769937f4501cc0e67f4f4483c8d2c3e1e922ee9edbe4ab4c7c0"
 [[package]]
 name = "discv5"
 version = "0.1.0-beta.3"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "f6f52d2228d51e8f868a37d5b5b25b82c13552b635d5b47c3a5d53855a6fc4f0"
+source = "git+https://github.com/sigp/discv5?rev=02d2c896c66f8dc2b848c3996fedcd98e1dfec69#02d2c896c66f8dc2b848c3996fedcd98e1dfec69"
```
Comment (michaelsproul) [inline Cargo.lock]: I think this change might have been caused by an old version of `cargo`, but we'll see if the Cargo.lock check on CI is happy

**Review by michaelsproul — APPROVED** [review]

**Review by michaelsproul — CHANGES_REQUESTED** [review]
Ah yeah, it failed (https://github.com/sigp/lighthouse/pull/2261/checks?check_run_id=2117512080)

If you update Rust to 1.50 and do `make lint` that should update `Cargo.lock` accordingly

**Comment by kirk-baird** [comment]
> Ah yeah, it failed (https://github.com/sigp/lighthouse/pull/2261/checks?check_run_id=2117512080)
> 
> If you update Rust to 1.50 and do `make lint` that should update `Cargo.lock` accordingly

Done :)

**Review by michaelsproul — APPROVED** [review]
Perfecto

bors r+

**Pull request closed without merging**